content/webapp/components/GuideStopCard/index.tsx

```diff
@@ -52,9 +52,8 @@ const GuideStopCard: FunctionComponent<Props> = ({
 }) => {
   const { viewTransitions } = useToggles();
 
-  const CardOuterComponent:
-    | typeof CardOuterTransition
-    | (typeof CardOuter & { href: string }) = viewTransitions
+  // TODO: constrain type
```

Comment:
For some reason I can't link to that section but in the TODO bit, we've gone for "no more TODOs". We either fix it on the spot or create a backlog ticket for it that we document in the PR. But not in code anymore!

Reply:
https://github.com/wellcomecollection/wellcomecollection.org/issues/11222